mempool/mempool.go

```diff
@@ -66,6 +66,9 @@ var (
 
 	// ErrMempoolIsFull means Tendermint & an application can't handle that much load
 	ErrMempoolIsFull = errors.New("Mempool is full")
+
+	// ErrPreCheck means the tx bytes are invalid - ie. too big.
+	ErrPreCheck = errors.New("Tx failed precheck")
```

Comment:
Should we be explicit about the reason in the msg / logs? Otherwise devs will have to dig into the code in order to figure out what precheck is.

Reply:
Yeh that would be nice - maybe those funcs should return errors instead of bool?

Reply:
Good idea